google-ads-examples/src/main/java/com/google/ads/googleads/examples/advancedoperations/AddAdCustomizer.java

```diff
@@ -133,6 +133,11 @@ public static void main(String[] args) throws IOException {
    */
   private void runExample(GoogleAdsClient googleAdsClient, AddAdCustomizerParams params) {
 
+    if (params.adGroupIds.size() != 2) {
```

Comment:
Rather than using 2, I would add as a constant called something like NUMBER_OF_AD_GROUPS under PAGE_SIZE. I would add a comment above the NUMBER_OF_ADS constant indicating that there should two ad groups because we're creating two ads, and each ad group has one ad.

Reply:
Right, thanks for spotting this. Done.